tests/responses.js

```diff
@@ -182,7 +182,9 @@ exports.getLoris = function(test) {
 };
 
 exports.getConnectionRefused = function(test) {
-	server.close();
+	try {
+		server.close();
+	} catch (e) { }
```

Comment:
Lets add a log here so that the error is not swallowed?

Reply:
@asturur II found the root cause of the change was because the file closes the server twice. moving this test into its own file solves the issue. https://github.com/shutterstock/armrest/commit/bf8fb8a91e0bf14af0ed49ee3da3a867d5bca95d 👍